### DISCUSSION TITLE
paraview cannot read the vtk file convected by the anim_to_vtk_win64.exe

### DISCUSSION THREAD

**Discussion by lzh-maker** [OpenRadioss/OpenRadioss#1485] [Q&A]
I used an executable file to convert the result file to vtk format, but the converted vtk file cannot be opened in Paraview, as shown in the following figure. Why is this?
![image](https://github.com/OpenRadioss/OpenRadioss/assets/76392842/40b8ec76-f4c8-4fca-9567-033a1b037e36)

**Comment by MarianBulla** [comment]
Hi Izh-maker,
can you try "Legacy VTK files" from "Files of type", please.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/9e711f44-d4b0-447a-af13-02077e57e2cb)

Thanks and best regards,
Marian

**Reply by lzh-maker** [reply]
[](url)Trying to open in the format "Legacy VTK", I got the following error
![image](https://github.com/OpenRadioss/OpenRadioss/assets/76392842/4b0d430e-2697-4b44-a233-25ba9fd6d506)
the vtk file is as [attached](url)
[C3D8_001.zip](https://github.com/OpenRadioss/OpenRadioss/files/11494761/C3D8_001.zip)

**Comment by sebastienVilleneuve** [comment]
Hi @lzh-maker 

can you link also the animation file you used ?

Thanks,
Sébastien

**Reply by PhiPsi-Software** [reply]
Do not use windows powershell to run anim_to_vtk_win64.exe.
